src/components/InlineCodeBlock/WrappedText.js

```diff
@@ -0,0 +1,87 @@
+import React, {Fragment} from 'react';
+import {Text, View} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../../styles/styles';
+
+/**
+ * Breaks the text into matrix
+ * for eg: My Name  is Rajat
+ *  [
+ *    [My,'',Name,'','',is,'',Rajat],
+ *  ]
+ *
+ * @param {String} text
+ * @returns {Array<String[]>}
+ */
+function getTextMatrix(text) {
+    return text.split('\n').map(row => row.split(/(\s)/));
+}
+
+const propTypes = {
+    // Required text
+    children: PropTypes.string.isRequired,
+
+    // Style to be applied to Text
+    // eslint-disable-next-line react/forbid-prop-types
+    textStyle: PropTypes.object,
```

Comment:
So we actually were just discussing this in [this PR](https://github.com/Expensify/Expensify.cash/pull/2185), @parasharrajat since you're such an active E.cash contributor I encourage you to look at it an get a sneak-peek at our new style guidelines.

But one of those guidelines we are going to standardize on is that all style props should take arrays, and be plural (aka `textStyles` instead of `textStyle`). That makes it easier to keep track of whether a style prop should be an object or an array, and prevents the component from having to accept either object or array props. If there's only one style, we'll just pass an array with a single object.

Does that make sense & can you implement that practice in this PR? Thanks! You can do `PropTypes.arrayOf(PropTypes.object))` too, which shouldn't need the `forbid-prop-types` suppression either.

Reply:
Wow. Great write-up. I like that the styles should be `Array<object>`. :+1: 

I will refactor it here.